packages/wallets/src/components/WalletCardIcon/WalletCardIcon.tsx

```diff
@@ -9,51 +9,53 @@ import GBP from '../../public/images/gbp.svg';
 import LTC from '../../public/images/ltc.svg';
 import USD from '../../public/images/usd.svg';
 import USDC from '../../public/images/usdc.svg';
+import './WalletCardIcon.scss';
 
 const typeToIconMapper = {
-    USD,
-    EUR,
-    GBP,
     BTC: Bitcoin,
-    USDC,
+    Demo,
     ETH,
+    EUR,
+    eUSDT: Tether,
+    GBP,
     LTC,
+    USD,
+    USDC,
     UST: Tether,
-    eUSDT: Tether,
-    Demo,
 };
 
-const typeToSizeMapper = {
-    USD: { mobile: 32, desktop: 48, small: 16 },
-    EUR: { mobile: 32, desktop: 48, small: 16 },
-    GBP: { mobile: 32, desktop: 48, small: 16 },
-    BTC: { mobile: 45, desktop: 90, small: 32 },
-    USDC: { mobile: 45, desktop: 90, small: 32 },
-    ETH: { mobile: 45, desktop: 90, small: 32 },
-    LTC: { mobile: 45, desktop: 90, small: 32 },
-    UST: { mobile: 45, desktop: 90, small: 32 },
-    eUSDT: { mobile: 45, desktop: 90, small: 32 },
-    Demo: { mobile: 45, desktop: 90, small: 32 },
+const typeToWidthMapper = {
+    rectangleIcon: { desktop: 90, mobile: 45 },
+    roundedIcon: { desktop: 48, mobile: 32 },
 };
 
+const typesWithRoundedIcon = ['EUR', 'GBP', 'USD'];
+
 type TProps = {
-    small?: boolean;
+    size?: 'lg' | 'md' | 'sm' | 'xs';
```

Comment:
can we make a generic type for this as we will be using it somewhere else too.
Just a suggestion `export type TSize = 'lg' | 'md' | 'sm' | 'xs';`
```suggestion
    size?: TSize;
```